author.js

```diff
@@ -70,6 +70,94 @@ admin_log_out_btn.addEventListener("click",()=>
 
 
 
+
+
+
+          function openAddAuthorForm() {
+
+            let input_addAuthor_form = document.querySelector("#add-author-form");
+            input_addAuthor_form.style.display = "block";
+            
+            input_addAuthor_form.innerHTML="";
+            
+            let heading = document.createElement('h1');
+            heading.innerText="Add New Author";
+            heading.style.textAlign="center";
+            
+            let authornameInput = document.createElement('input');
+            authornameInput.className="take-input";
+            authornameInput.type = 'text';
+            authornameInput.required = true;
+            authornameInput.placeholder="Enter Name";
+
+
+            let authornationalityInput = document.createElement('input');
+            authornationalityInput.className="take-input";
+            authornationalityInput.type = 'text';
+            authornationalityInput.required = true;
+            authornationalityInput.placeholder="Enter Nationality";
```

Comment:
Create a separate function to generate input field and use that function everywhere it needed in the app.